ai-service/tests/conftest.py

```
@@ -16,16 +16,19 @@ def create_db_test_collection(collection_name: str) -> chromadb.Collection:
+    collection_name: str = f"{module_name}"
     return create_db_test_collection(collection_name)
 
 
 @pytest.fixture(autouse=True)
 def patch_and_clean_db_collection(
-    monkeypatch, db_test_collection
+    monkeypatch: pytest.MonkeyPatch,
+    db_test_collection: chromadb.Collection,
 ) -> Generator[None, None, None]:
-    monkeypatch.setattr(db, "collection", db_test_collection)
+    monkeypatch.setattr("ai_service.db.get_collection", lambda name: db_test_collection)  # type: ignore[arg-type]
```

Ruff (0.12.2)

Unused lambda argument: `name`

(ARG005)